static/includes/apps/AppsDatasets.md

```diff
@@ -8,6 +8,8 @@ Any modification of it can result in instability and loss of app functionality.
 
 Do not include the ix-apps dataset inside of an SMB or NFS share.
 
+The ix-apps dataset does not inherit encrpytion if an encryprted pool is configured for applications.
```

Comment:
Are you saying if the pool selected for apps is encrypted? Or, if they are purposely encrypting a pool for apps? Subtle difference but which is more likely?

Reply:
If the pool in Configuration > Choose pool is encrypted, ix-apps ignores this and is created as an unencrypted dataset. Is this clearer?

```suggestion
The ix-apps dataset does not inherit encryption if an encrypted pool is selected as the pool for applications.
```